### ISSUE TITLE
Contradictory Information when using Ollama phi3:medium

### ISSUE THREAD

**Issue by zwei2016** [ScrapeGraphAI/Scrapegraph-ai#844]
**Describe the bug**
A contradictory information is found when loading the ollama/phi3:medium model.

```
    "llm": {
        "model": "ollama/phi3:medium",
        "temperature": 0,
        "format": "json",  # Ollama needs the format to be specified explicitly
        "base_url": "http://localhost:11434",  # set Ollama URL
    },
```

```
Model ollama/phi3:medium not found,
                    using default token size (8192)
```

However, the program seems work and return the web page content with the model.

**To Reproduce**
1: use the ollama/phi3:medium in llm config
2: run it

**Expected behavior**
the phi3:medium should be recognized by config. The main reason probably is that this model is not listed in 
scrapegraphai/helpers/models_tokens.py

**Screenshots**

![image](https://github.com/user-attachments/assets/9f9b7a0a-288c-493f-b02d-39e57d356f59)

**Desktop (please complete the following information):**
 - OS: Win11
 - Browser: Opera
 - Version: 114.0.5282.235

**Comment by PeriniM** [comment]
Yes, you are right! You can bypass it by specify the `model_tokens`param like:

```python
graph_config = {
    "llm": {
        "api_key": os.getenv("OPENAI_API_KEY"),
        "model": "ollama/phi3:medium",
        "model_tokens": 10000,
    },
    "verbose": True,
    "headless": False,
}
```

We will update the logging
